src/cpp/rtps/transport/test_UDPv4Transport.h

```diff
@@ -60,6 +60,9 @@ class test_UDPv4Transport : public UDPv4Transport
     RTPS_DllAPI static bool always_drop_participant_builtin_topic_data;
     RTPS_DllAPI static bool simulate_no_interfaces;
 
+    typedef std::function<bool (const Locator& destination)> DestinationLocatorFilter;
+    RTPS_DllAPI static DestinationLocatorFilter locator_filter;
```

Comment:
Could this filter be non-static like `drop_data_frag_messages_filter_`?

Reply:
Or have this static one and another non-static

Reply:
8c544e4